.github/workflows/stack-deployment.yaml

```diff
@@ -41,7 +41,12 @@ jobs:
 
       - name: Set CDK output
         working-directory: cdk
-        run: echo "CDK_OUTPUT=$(cat cdk-outputs.json | jq '.JsAssertionCheatSheet')" >> $GITHUB_ENV
+        run: |
+          {
+            echo 'CDK_OUTPUT<<EOF'
+            $(cat cdk-outputs.json | jq '.JsAssertionCheatSheet')
```

Comment:
Did you try `jq -c`?

Reply:
Nope, but it seems hat should work as it deletes newlines. Thanks!